snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/compose/ComposeSnapshotter.kt

```diff
@@ -120,8 +143,8 @@ private fun measureComposableSize(
     previewConfig.heightDp?.let { View.MeasureSpec.UNSPECIFIED } ?: View.MeasureSpec.AT_MOST
 
   view.measure(
-    View.MeasureSpec.makeMeasureSpec(max(view.width, widthDpPx), widthMeasureSpec),
-    View.MeasureSpec.makeMeasureSpec(max(view.height, heightDpPx), heightMeasureSpec),
+    View.MeasureSpec.makeMeasureSpec(max(view.width, widthPx.toInt()), widthMeasureSpec),
```

Comment:
I moved the integer rounding to after the px have been calculated. We were losing precision on the density.